shared/lib/gator-permissions/gator-permissions-utils.ts

```diff
@@ -1,4 +1,9 @@
-import { Hex } from '@metamask/utils';
+import {
+  Hex,
+  hexToNumber,
+  KnownCaipNamespace,
+  CaipChainId,
```

Comment:
These should be imported as type
```suggestion
  type Hex,
  hexToNumber,
  type KnownCaipNamespace,
  type CaipChainId,
```
or alternatively have two separate imports - one for type, one for function import